jerry-core/parser/js/js-parser-expr.c

```diff
@@ -281,6 +282,11 @@ parser_parse_array_literal (parser_context_t *context_p) /**< context */
 
       parser_parse_expression (context_p, PARSE_EXPR_NO_COMMA);
 
+      if (context_p->last_cbc_opcode == CBC_PUSH_THIS)
```

Comment:
Shouldn't be this inside the `if (context_p->token.type == LEXER_COMMA)` condition?

Reply:
Hm, probably irrelevant where it is. Ok, let it be this way.

Reply:
Good idea.